uws/UWS/models.py

```diff
@@ -10,6 +10,25 @@ def _parse_bool(self, value):
             return False
         return value
 
+    def _find_ns_uws(self, namespaces):
+        # Search for namespace that contains "http://www.ivoa.net/xml/UWS/"
+        # (usually, there is a version attached, which we ignore here),
+        # for using it (including {}) as uws_ns in the
+        # find/findall-functions everywhere below.
+        ns = None
+        for p in namespaces.keys():
+            value = namespaces[p]
+            if 'http://www.ivoa.net/xml/UWS/' in value:
+                ns = value
+
+        if ns is None:
+            "Warning: no matching UWS namespace found in xml-response, probably won't find anything."
+            uws_ns = ''
+        else:
+            uws_ns = '{' + ns + '}'
+        
+        return uws_ns
+
```

Comment:
You could actually write a unittest for this method, making sure this actually works.